docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+        <title>Sasha</title>
+
+        <para>Sasha is an outlier as it is a dictionary type structure and not
+        a list. You can't have more than one archiver or dfuwu-archiver as
+        that is a value limitation, you can choose to either have the service
+        or not (set the 'disabled' value to true). The Sasha component is an
+        exception to the norm for HPCC Systems components.</para>
+      </sect3>
+
+      <sect3 id="ValYML_Thor">
+        <title>Thor</title>
+
+        <para>In the case of Thor (components) there are similarities with the
+        ECLCC Server. Thor is also a list, and if that list has a value
+        of</para>
+
+        <programlisting>-name: thor </programlisting>
+
+        <para>Or</para>
+
+        <programlisting>-name: thor_50</programlisting>
+
+        <para>If you have a small job you don't need 400 nodes when 10 would
+        do. If you wanted 50 nodes set the number of workers the <emphasis
+        role="bold">numWorkers</emphasis> value to 50 and you would have 50
+        nodes. In a case when multiple Thors would be useful:</para>
+
+        <para><programlisting>thor:
+- name: thor
+  prefix: thor
+  numWorkers: 2</programlisting></para>
+
+        <para>Continuing on this example, if you wanted a 50 node Thor, you
+        set <emphasis>numWorkers:</emphasis> value to 50 and that would deploy
+        a 50 node Thor. In doing so, ideally you should rename the resource to
+        something like <emphasis>thor_50</emphasis> as in the above example,
+        and that would then re-read the file and spawn the 50 node Thor.
+        Caution however, if you were to put that in a helm update that would
+        do nothing, if there are no jobs running there are no Thor instances
+        running.</para>
+
+        <para>Similarly to the <emphasis>maxActive</emphasis> values in the
+        ECLCC Server, Thor has some analogous options.</para>
+
+        <para><emphasis role="bold">MaxJobs</emphasis>: controls the number of
+        jobs, specifically <emphasis>MaxJobs</emphasis> sets maximum number of
+        jobs.</para>
+
+        <para><emphasis role="bold">MaxGraphs</emphasis>: likewise
+        <emphasis>MaxGraphs</emphasis> also limits the maximum amount of
+        graphs. It makes sense to keep this value at same number, since not
+        all jobs submit graphs - but some jobs may be contained in ECL Agent
+        and may not want Thor graphs. In that case it wouldn't make sense to
+        have max of 4 If you need a Thor to generate graphs. If there are more
+        than 2 submitted (Thor) graphs, the second would be blocked until the
+        next Thor instance becomes available. It may simply make sense to set
+        <emphasis>maxGraphs</emphasis> to the same value as
+        <emphasis>MaxJobs</emphasis>.</para>
+
+        <para>The delivered (example) chart values defines maxJobs to be
+        greater than maxGraphs. It does so, because that makes sense. Jobs
```

Comment:
see prev. comment. Looks like this was based on previous feedback, but the previous section which states it makes senss for them to be same needs to be revised or deleted.